Web/starter_project_iconics/components/layout/Header.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { MdOutlineClose, MdViewHeadline } from 'react-icons/md'
+import logo from '../../public/a2sv-logo.png'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+const Header = () => {
+  const [navigation, setNavigation] = useState(() => [
+    { id: 0, name: 'Home', to: '/', current: true },
+    { id: 1, name: 'Teams', to: '/team/teams', current: false },
+    {
+      id: 2,
+      name: 'Success Stories',
+      to: '/Success_story/success_stories',
+      current: false,
+    },
+    { id: 3, name: 'About Us', to: '/about/about', current: false },
+    { id: 4, name: 'Blogs', to: '/blog/blogs', current: false },
+    {
+      id: 5,
+      name: 'Get Involved',
+      to: '/get_involved/get_involved',
+      current: false,
+    },
+  ])
+
+  const [showNav, setShowNav] = useState(false)
+
+  const handleNavClick = (e: React.FormEvent) => {
+    let id = parseInt((e.target as HTMLAnchorElement).id)
+
+    setNavigation(
+      navigation.map((item) => {
+        if (item.id == id) {
+          return { ...item, current: true }
+        } else {
+          return { ...item, current: false }
+        }
+      })
+    )
+  }
+
+  return (
+    <div className="bg-white flex justify-between p-5">
+      <div className="w-28 lg:w-52">
+        <Image
+          src={logo}
+          alt="a2sv-logo"
+          width={200}
```

Comment:
i recommend only setting one (width or height) and letting the image scale on it's own (this would make it easier to make changes and won't distort or crop the image)
and i don't think we want cover for this logo as we want it to sit as it is